examples/QuickStart/JavaSpring/QuickStart/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
```

Comment:
Why does JavaSpring folded have a .gitignore file? Other languages don't have it.